src/components/styles/nav.css

```diff
@@ -0,0 +1,247 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600&display=swap');
+
+.nav-container {
+  height: 5.938rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+}
+
+.navtitle a {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+  text-decoration-line: none;
+}
+
+.icon-img {
+  width: 3rem;
+  height: 3rem;
+  margin: 0 0 0 55rem;
+  padding: 0.875rem;
+  color: #0290ff;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: kindly align the icon on the navbar to the right in a straight line with the cards just as it's in the original design. Doing that will make your design looks organized and neat. Check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
| ![Screenshot from 2023-05-28 04-02-41](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/86bae9fb-5678-46d9-b220-b0ea0239dc49) |![Screenshot from 2023-05-28 04-02-59](https://github.com/emmanuelsteven/BookStore-React-Redux/assets/45226848/28db0ac7-765b-460c-bf4b-602bcf9f9118) |